sycl/source/detail/scheduler/graph_processor.cpp

```diff
@@ -23,16 +23,17 @@ static Command *getCommand(const EventImplPtr &Event) {
 
 std::vector<EventImplPtr>
 Scheduler::GraphProcessor::getWaitList(EventImplPtr Event) {
-  Command *Cmd = getCommand(Event);
```

Comment:
Could you please add a test to sycl/unittests?

Reply:
I added sycl/unittests/event with test for it